scm/src/scm_input.F90

```diff
@@ -46,7 +46,7 @@ subroutine get_config_nml(scm_state)
   integer              :: n_snow   !< number of model snow levels (currently only 3 supported)
   integer              :: n_columns !< number of columns to use
   integer              :: n_time_levels
```

Comment:
`n_time_levels` controls the size of some state arrays, but if we're getting rid of the only time scheme that required more than one time level, we can get rid of this too. Of course, it would involve going through the code and removing a dimension from the state variables. Is the simplification worth the work?